src/Driver/PHPDBG.php

```diff
@@ -99,6 +99,9 @@ private function detectExecutedLines(array $sourceLines, array $dbgData): array
             }
         }
 
-        return $sourceLines;
+        return [
```

Comment:
This doesn't match the contract provided in #13

Reply:
I do think it's fine that the transformation is applied here, though.

Reply:
...Oops. This commit brought to you by lack of ☕️